src/CalculatorUITestFramework/HistoryPanel.cs

```diff
@@ -6,26 +6,41 @@
 using OpenQA.Selenium.Appium;
 using OpenQA.Selenium.Appium.Windows;
 using System.Collections.ObjectModel;
+using System.Drawing;
+using OpenQA.Selenium.Interactions;
+using System;
```

Comment:
Please sort your using statements.  If you're using visual studio to code this up, you can right click this and select "remove and sort usings".

Reply:
I will

Reply:
Done